minio/policy.py

```diff
@@ -24,12 +24,12 @@
 
 """
 
-from .compat import basestring
-
 import collections
 import fnmatch
 import itertools
 
+from .compat import basestring
```

Comment:
can you update the licence header for these files as well pointing to '2015, 2016'.